concert/tests/integration/test_imaging_experiments.py

```diff
@@ -324,34 +405,68 @@ def setUp(self):
                                                       num_darks=2,
                                                       num_projections=15)
         self.writer = ImageWriter(walker=self.walker, acquisitions=self.exp.acquisitions)
+        self.acc = Accumulator(self.exp.acquisitions)
+
+    async def test_number_of_frames(self):
+        if (await self.exp.get_iteration()) < 2:
```

Comment:
What is this for?

Reply:
This is no longer there. Here I tried to convince my local test setup to run the experiment only once.